docs/src/components/Sidebar/Sidebar.tsx

```diff
@@ -38,6 +39,14 @@ class Sidebar extends React.Component<any, any> {
   _searchInput: any
   selectedRoute: any
   filteredMenu = componentMenu
+  treeRef: Ref
+  searchInputRef: Ref
```

Comment:
Oops, it should be

```suggestion
  searchInputRef: React.RefObject<HTMLElement>
```

However, you don't need `constructor()` 🐱 
```suggestion
  searchInputRef: React.createRef()
```